client/js/ifml2code/client/elementrules/templates/system-event-vm.js.ejs

```diff
@@ -0,0 +1,51 @@
+/*jslint node: true, nomen: true */
+"use strict";
+
+var ko = require('knockout');
+
+function ViewModel(params) {
+    var self = this;
+
+    // TODO: Execution
+    /*
+      example:
+        //to trigger the event: params.fire;
+        document.addEventListener('keydown', params.fire);
+    */
+
+    self.context = params.context;
+
+    // THIS CAN BE REMOVED (BEGIN)
+    self.stop = function () {
+      // TODO: Execution
```

Comment:
Give a better description for the TODO.
Maybe TODO: Teardown everything you setup to listen for the event